cpp/src/centrality/betweenness_centrality.cu

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
We need a source file to compile the object into the library.  Since there are no kernels, I could make it a .cpp file.  This would compile slightly faster and matches the technique used in cuDF.

Reply:
We don't do that too much in cugraph at this point.